src/main/java/com/everis/boundary/JAXRSConfiguration.java

```diff
@@ -0,0 +1,48 @@
+package com.everis.boundary;
+
+import java.util.Arrays;
+
+
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import javax.ws.rs.ApplicationPath;
+import javax.ws.rs.core.Application;
+
+import io.swagger.v3.jaxrs2.integration.JaxrsOpenApiContextBuilder;
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.integration.OpenApiConfigurationException;
+import io.swagger.v3.oas.integration.SwaggerConfiguration;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.servers.Server;
+
+@ApplicationPath("/api")
+@OpenAPIDefinition(info = @Info(title = "Everis-Car-App_API", version = "0.0", description = "Car's CRUD Functionality"))
+public class JAXRSConfiguration extends Application  {
+	
+	@SuppressWarnings("rawtypes")
+	public JAXRSConfiguration() {
+        super();
+        OpenAPI oas = new OpenAPI();
+        oas.setServers(null);
+        Server server = new Server();
+        server.setUrl("http://localhost:8080/car-api");
```

Comment:
About our discussion this morning, we agreed on changing the hard-coded server address and port to something variable. Could you please check it ?